modules/acquia_cms_tour/src/Plugin/AcquiaCmsStarterKit/StarterKitConfigForm.php

```diff
@@ -204,7 +209,7 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
       $this->state->set('show_starter_kit_modal', FALSE);
       $this->state->set('starter_kit_wizard_completed', TRUE);
       $service->enableModules($starter_kit, $starter_kit_demo, $starter_kit_content_model);
-      $this->messenger()->addStatus('The required starter kit has been installed. Also, the related modules & themes have been enabled.');
+      $this->messenger()->addStatus('The ' . $starter_kits[$starter_kit] . ' starter kit has been installed. Also, the related modules & themes have been enabled.');
```

Comment:
@rajeshreeputra use $this->messenger()->addStatus($this->t('The  %starter_kit starter kit has been installed. Also, the related modules & themes have been enabled.', ['%starter_kit' => $starter_kits[$starter_kit]]))

Reply:
done